### DISCUSSION TITLE
Discord

### DISCUSSION THREAD

**Discussion by RWOverdijk** [founded-labs/react-native-reusables#230] [Ideas]
I don't know how others feel about it, but I think having a discord would be nice.

I've tried every single ui library under the sun and the only good one I ever used was mantine, which does not work for mobile. I think nrn could benefit from a place for the community to gather and maybe even cooperate to work on third party reusables.

**Comment by mrzachnugent** [comment]
Hey @RWOverdijk, we have a #react-native-reusables channel in the [rõnin Discord](https://discord.gg/fSfgzNTQ). We'd love to have you join us.
